docs/self-managed/operational-guides/multi-region/dual-region-ops.md

```diff
@@ -141,17 +140,17 @@ desired={<Five viewBox="140 40 680 500" />}
 | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | You have ensured that you fully lost a region and want to start the temporary recovery. <br /> <br /> One of the regions is lost, meaning Zeebe: <br /> - No data has been lost thanks to Zeebe data replication. <br /> - Is unable to process new requests due to losing the quorum <br /> - Stops exporting new data to Elasticsearch in the lost region <br /> - Stops exporting new data to Elasticsearch in the survived region | The lost brokers have been removed from the Zeebe cluster. <br /> <br /> Continued processing is enabled, and new brokers in the failback procedure will only join the cluster with our intervention. |
 
-#### How to get there
+#### Procedure
 
-You will port-forward the `Zeebe Gateway` in the surviving region to the local host to interact with the Gateway.
+Start with creating a port-forward to the `Zeebe Gateway` in the surviving region to the local host to interact with the Gateway.
 
 The following alternatives to port-forwarding are possible:
 
-- if it's exposed to the outside, one can skip port-forwarding and use the URL directly
-- one can [`exec`](https://kubernetes.io/docs/reference/kubectl/generated/kubectl_exec/) into an existing pod (such as Elasticsearch), and `curl` from there
-- or temporarily [`run`](https://kubernetes.io/docs/reference/kubectl/generated/kubectl_run/) an Ubuntu pod in the cluster to `curl` from there
+- if the gateway is not exposed to the outside of the Kubernetes cluster, you can skip port-forwarding and use the URL directly
```

Comment:
the contrary is the case

if exposed --> use URL
if not --> port-forwarding

not exposed --> can't use URL